R/app_ui.R

```diff
@@ -24,6 +24,11 @@ app_ui <- function() {
         projectlive.modules::summary_snapshot_module_ui("summary_snapshot_ui_1"),
         icon = shiny::icon("chart-area")
       ),
+      shiny::tabPanel(
+        "Studies",
+        projectlive.modules::study_summary_module_ui("study_summary_ui_1"),
```

Comment:
You may need to modify the y-axis limits on the publication plot to be accomodative of >5 publications :)

Reply:
@jaybee84 https://github.com/Sage-Bionetworks/projectlive.modules/blob/7a4a6325f547f255ec15f34f4c8d1d4eac74ef9c/R/plot_functions.R#L43

Reply:
@jaybee84 All plots using create_plot_with_config() have plotly::layout(autosize =T) added to them.

Reply:
I've fixed y-axis issues. The plot function had y-axis limit of 10 in one spot, and 5 in another. I'm not sure how those got there, maybe they were in the original code I used to create them, and just got left in?

Reply:
hmm.. it weird that even with layout:: autosize , the resources generated plot is skewed to one side of the tabpanel box... I know we encounted this earlier too but AFAIK the layout had resolved it. 

Any idea why this is happening for that one plot? The same function seems to work well for projectLive-NF though..

Reply:
Also, can you remove the unannotated (highest bar) from the `resources generated` plot? That would help us visualize the other bars better

Reply:
Not sure about the plots. I've given up trying to get ggplotly plots to layout correctly. :)

Reply:
The publication status plot in the studies module still has the yaxis limits 0-5

Reply:
oh wait I might need to update my projectlive.modules

Reply:
OK I've made some changes, please try them.

Reply:
Looks great!

Did you do something to modify the `resources_generated` plot? It looks like it is working well now.